thirdeye/thirdeye-frontend/app/pods/components/yaml-editor/component.js

```diff
@@ -137,6 +281,116 @@ export default Component.extend({
     }
   }).drop(),
 
+  didRender(){
+    this._super(...arguments);
+
+    later(() => {
+      this._buildSliderButton();
+    });
+  },
+
+  // Helper function that builds the subchart region buttons
+  _buildSliderButton() {
+    const componentId = this.get('componentId');
```

Comment:
feel like this could be in a util. It might also exists already. We can take off line.